app/views/subnets/_bootdisk_title_buttons.erb

```diff
@@ -0,0 +1 @@
+<%= display_bootdisk_title_buttons %>
```

Comment:
I wonder if the code for both templates could actually live here and you would get rid of an extra helper. But I am not a RoR expert, not sure if that's not a good practice. Sounds reasonable to me tho.

Reply:
I learned that every unique piece of code should be in project only once (Don't repeat yourself - DRY). So since these helper has very similar codebase, it makes sense to move same functionality to another place and you just include them in desired helpers. 

So I can get rid of extra helper but there will be almost same code twice and I don't think it's a good idea. What do you think?

Reply:
Okay DRY is good sure.